doc/source/ray-observability/key-concepts.rst

```diff
@@ -77,7 +77,8 @@ Profiling is a way of analyzing the performance of an application by sampling th
 
 - CPU profiling for Driver and Worker processes, including integration with :ref:`py-spy <profiling-pyspy>` and :ref:`cProfile <profiling-cprofile>`
 - Memory profiling for Driver and Worker processes with :ref:`memray <profiling-memray>`
-- GPU profiling with :ref:`Pytorch Profiler <profiling-pytorch-profiler>` and :ref:`Nsight System <profiling-nsight-profiler>`
+- GPU profiling with :ref:`Pytorch Profiler <profiling-pytorch-profiler>`, :ref:`Nsight System <profiling-nsight-profiler>`
+- GPU profiling with :ref: `Unitrace Profiler <profiling-unitrace-profiler>` for Intel(R) GPUs
```

Comment:
The extra space after `:ref:` needs to be removed otherwise the link won't be recognized.